Source/Private/Convert-ToDockerHostPath.ps1

```diff
@@ -0,0 +1,24 @@
+function Convert-ToDockerHostPath {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        [String]
+        $Path
+    )
+
+    $pathOnDockerHost = $Path
+    if ( (Invoke-DockerCommand 'ps').StdOut | Select-String $(hostname) ) { 
```

Comment:
This line produces some noise in the logs. Perhaps it can be executed without having anything printed out.
```suggestion
    if ( (Invoke-DockerCommand 'ps').StdOut | Select-String $(hostname) | Out-Null ) { 
```
I have not tested the suggestion.

Reply:
May be I'm wrong, but I do not see how additional logging is produced here. No noise is expected. And proposed change will break the logic ('if' condition will never be True).

Reply:
I'll find a way to silence this part out.

Reply:
fixed with -Quiet switch